backends/bmv2/expression.cpp

```diff
@@ -201,9 +188,9 @@ void ExpressionConverter::postorder(const IR::Member* expression)  {
     if (parentType->is<IR::Type_StructLike>()) {
         auto st = parentType->to<IR::Type_StructLike>();
         auto field = st->getField(expression->member);
-        if (field != nullptr)
-            // field could be a method call, i.e., isValid.
-            fieldName = field->controlPlaneName();
+        BUG_CHECK(field != nullptr,
```

Comment:
There are other methods that look like fields, e.g., push.
So maybe this should not be a BUG.